### DISCUSSION TITLE
Evil snipe

### DISCUSSION THREAD

**Discussion by joelreymont** [zed-industries/zed#19886] [Feature Requests]
I sorely miss the [evil snipe](https://www.youtube.com/watch?v=ywRExNOyybY) feature [available](https://github.com/hlissner/evil-snipe) for Emacs. 

Where should I start to implement it?

How difficult would this be, i.e. does the editor architecture provide for this?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
